src/js/dp/release-calendar.js

```diff
@@ -30,3 +36,118 @@ document.addEventListener("DOMContentLoaded", function () {
     releaseTypeAutoSubmit(".release-calendar__filters .filters__release-type");
   }
 });
+
+const sortSelector = document.querySelector(".ons-input--select");
+if (sortSelector) {
+  sortSelector.addEventListener("change", async (e) => {
+    gtmDataLayerPush({
+      event: "SortBy",
+      "sort-by": e.target.value,
+    });
+  });
+}
+
+if (releaseCalendarContainer) {
+  [
+    ...releaseCalendarContainer.querySelectorAll(
+      ".ons-radio__input[type=radio]:not(input:disabled)"
+    ),
+  ].map((topFilter) => {
+    topFilter.addEventListener("change", async (e) => {
+      gtmDataLayerPush({
+        event: "Filter",
+        "filter-by": e.target.name,
+        selected: e.target.value.replace("type-", ""),
```

Comment:
Probably worth checking that e.target.value is indeed a string before calling replace on it so it doesn't crash when it's not a string, and providing an alternative value otherwise

Reply:
Or cast that value into a string so the replace always succeeds, either way...

Reply:
done.

Reply:
My original comment concerned`e.target.value`, not `e.target.name`. Please redo this.

Reply:
done